lib/Corres_UL.thy

```diff
@@ -1256,6 +1256,18 @@ lemma corres_stateAssert_implied2:
   apply simp
   done
 
+lemma corres_stateAssert_add_assertion:
```

Comment:
not sure what we ended up agreeing on tagging commit messages to `lib/`, but this commit has the generic form of `lib: add something` that we discussed previously

Reply:
unless this is the commit you refer to that's already in another PR, in which case please ignore

Reply:
This commit is from another PR, but if there is an issue with the commit message, then I guess it needs to be fixed there. 

I thought the issue with `lib: add blah` was that we might not necessarily know "where" in `lib` we were adding the rule to. I did ask for some clarification on this in the guide we have on commit messages, but this hasn't happened yet. I thought that in this case, since one would assume (correctly, in this case) that a lemma related to `corres` would be in `Corres_UL` or `ExtraCorres`, further specificity wasn't needed here. If the current commit message isn't OK, could you suggest a better one please?

Reply:
OK, I looked though a while of history of `Corres_UL.thy` and yeah, you have a point. Avoiding the by-the-way commits (changes to files based on some other goal than adding a rule), they oscillate between doing what you did, and adding a `to Corres_UL` at the end. I have a minor preference for the latter, because it could be one of two files, but it's definitely survivable if you don't change it.
I'm trying to steer you towards documenting stuff by default, but in my efforts some of the cases might be too trivial. Apologies on this one.